### ISSUE TITLE
Photomap (Report) screen needs namefinder

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/ios#83]
The namefinder present in the journey planning tab could very usefully be added to the Photomap (Report) screen too.

Ideally it would also be present within the Wizard on the "If you want to change the chosen photo's location" part.

I often find myself needing this, and instead end up searching in the first screen and doing map-matching across the two tabs as a poor workaround.

**Comment by mvl22** [comment]
It would be good if we could get this into the next release - I keep running into this. Presumably this is not a difficult thing to add?

**Comment by neilkachu** [comment]
Yes can be done as this is a modal dialog anyway and it returns a coordinate to the invoking controller.
